src/eventDispatchers/mouseEventHandlers/mouseMove.js

```diff
@@ -57,6 +75,12 @@ export default function (evt) {
         data.active = !data.active;
         imageNeedsUpdate = true;
       }
+
+      // TODO: We may want to do _only_ this, and provide a default implementation
+      // TODO: On each tool?
+      if (typeof tool.mouseMoveCallback === 'function') {
+        tool.mouseMoveCallback(evt);
```

Comment:
Do we need an `active` version of this callback as well? One that tools without data can utilize? 🤔

Reply:
Feel free to disagree, but I'm an advocate of not adding functionality that isn't currently needed. If a current tool needs this then sure. If its needed for a future tool we can make a new PR for it.

I'm even inclinded to remove these few lines, as I don't think we need it now? mouseMove is only used for hover activation, and custom cursors are now supported without the tool needing a callback.